bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -1723,7 +1723,66 @@ StringRef getSectionName(const SectionRef &Section) {
   return Name;
 }
 
-// Extracts an appropriate slice if input is DWP.
+// Exctracts some appropriate slices of .debug_str.dwo from DWP.
```

Comment:
Please update description to be more descriptive of what "appropriate" means.
Also maybe couple of sentences on overall algorithm, and mention Fragment optimization, and saving writes on offsets if they are the same.